theano/tensor/nnet/corr_gemm.c

```diff
@@ -220,8 +239,10 @@ PyArrayObject* corrMM(PyArrayObject* bottom,
     const int N_ = col_dim[2];
     const int col_stride = (K_ * N_);
     const int M_ = nFilters;
+    const int one_int = 1;
     const %(c_float_type)s one = 1.0;
     const %(c_float_type)s zero = 0.0;
+    const int ldw = (K_ * N_);
```

Comment:
I thought it wouldn't make sense to use `col_stride` to indicate the leading dimension of weights, so a separate variable would be needed for clarity.